### PR TITLE
DEVDOCS-3803: Update payments-api-overview

### PR DESCRIPTION
**Pull request by traci707**
# [DEVDOCS-3803](https://jira.bigcommerce.com/browse/DEVDOCS-3803)

## What changed?
Updated Get payment methods response

### CODE REVIEW

File: docs/api-docs/payments/payments-api-overview.md [hunk 1]
```diff
@@ -264,6 +280,11 @@ Content-Type: application/json
 
 If the purchase was successful, the response returns a status of success. The order is then automatically moved to an Awaiting Fulfillment status. If you get a different response, see [Error codes](#error-codes) for troubleshooting.
 
+In the case of store credit and gift certificates:
+* If store credit and/or gift certificate covers the entire order amount, the order will be moved to an Awaiting Fulfillment status.
+* The order will stay in Pending status until it is fully paid. You can make the remaining order payment using other payment methods (Credit card, stored card, or store PayPal account) in the next payment request.
+
+For more information, see [Store Credit/Gift Certificate usage in S2S APIs](https://bc-eng-docs.herokuapp.com/domains/payments/technical-initiatives/s2s-store-credit-gift-certificate-usage.html).
```
Comment (bc-harika) [inline docs/api-docs/payments/payments-api-overview.md:287]: Could you please remove the docs link. It is an internal doc, had shared only for your reference.

File: docs/api-docs/payments/payments-api-overview.md [hunk 2]
```diff
@@ -264,6 +280,10 @@ Content-Type: application/json
 
 If the purchase was successful, the response returns a status of success. The order is then automatically moved to an Awaiting Fulfillment status. If you get a different response, see [Error codes](#error-codes) for troubleshooting.
 
+In the case of store credit and gift certificates:
+* If store credit and/or gift certificate covers the entire order amount, the order will be moved to an Awaiting Fulfillment status.
+* The order will stay in Pending status until it is fully paid. You can make the remaining order payment using other payment methods (Credit card, stored card, or store PayPal account) in the next payment request.
```
Comment (bc-harika) [inline docs/api-docs/payments/payments-api-overview.md:285]:
```suggestion
* The order will stay in Pending status until it is fully paid. You can make the remaining order payment using other payment methods (credit card, stored card, or store PayPal account) in the next payment request.
```
May be remove the capitalisation of `Credit card` inside the parentheses?

**Review by bc-harika — APPROVED** [review]
Looks good, just had a minor comment.

Please hold off publish the docs until we release the feature. Will let you know once we are ready 🙇

**Comment by bc-harika** [comment]
Not part of changes done in this PR, but there is a text change that needs to be fixed in this file. Happy to create a separate ticket if needed.

`Example request: Process payment and save PayPal account` should change to `Example request: Process payment with a stored PayPal account`

**Comment by traci707** [comment]
@bc-harika thank you for reviewing. I created a separate PR for the small change you requested [PR 1487](https://github.com/bigcommerce/dev-docs/pull/1487). We will not merge this PR until we hear from you.

**Review by bc-tgomez — DISMISSED** [review]
LGTM!

**Review by slsriehl — APPROVED** [review]
LGTM

**Pull request merged**